markdown/index.html

```diff
@@ -19,16 +19,16 @@ <h1>Markdown editor</h1>
 	</div>
 	<div class="editor-wrapper">
 		<div id="cke5-markdown-demo">
-## Markdown output 🛫
+			## Markdown output 🛫
```

Comment:
If Markdown code is indented, the editor thinks it's a codeblock:

![image](https://user-images.githubusercontent.com/10883184/228544912-03b751bc-dc0b-4e90-a09d-a45282d2c891.png)

If should be outdented like the rest of the content in this demo.

Reply:
I can still reproduce this issue:
![image](https://user-images.githubusercontent.com/10883184/229760390-95b828f0-48d9-4109-8b2f-758a7715e54d.png)